docs/docs/integrations/providers/unstructured.mdx

```diff
@@ -42,30 +47,21 @@ Check out the instructions
 
 ## Data Loaders
 
-The primary usage of the `Unstructured` is in data loaders.
-
-### UnstructuredAPIFileIOLoader
-
-See a [usage example](/docs/integrations/document_loaders/unstructured_file#unstructured-api).
-
-```python
-from langchain_community.document_loaders import UnstructuredAPIFileIOLoader
-```
+The primary usage of `Unstructured` is in data loaders.
 
-### UnstructuredAPIFileLoader
+### UnstructuredLoader
 
-See a [usage example](/docs/integrations/document_loaders/unstructured_file#unstructured-api).
+See a [usage example](/docs/integrations/document_loaders/unstructured_file) to see how you can use
+this loader for both partitioning locally and remotely with the serverless Unstructured API.
 
 ```python
-from langchain_community.document_loaders import UnstructuredAPIFileLoader
+from langchain_unstructured import UnstructuredLoader
 ```
 
 ### UnstructuredCHMLoader
 
 `CHM` means `Microsoft Compiled HTML Help`.
 
-See a usage example in the API documentation.
```

Comment:
I did not see an exiting example in the API documentation, so I removed this to not mislead users.